.github/actions/reportportal-prepare/action.yml

```
@@ -17,6 +17,11 @@ inputs:
   rp-extra-attributes:
     description: Extra attributes for Report Portal
     required: false
+  rp-static-name:
```

```suggestion
  rp-static-launch-name:
```

A more descriptive name might help.